packages/orbit-components/src/Itinerary/ItinerarySegment/index.tsx

```diff
@@ -25,6 +25,10 @@ const ItinerarySegment = ({
   const [opened, setOpened] = React.useState(false);
 
   const handleClick = (ev: React.SyntheticEvent<HTMLDivElement>) => {
+    const target = ev.target as HTMLElement;
+    const isFromScroll = target.closest(".orbit-horizontal-scroll");
```

Comment:
Could we rename this variable to something more understandable, such as `isHorizontalScroll`?